docs/source/markdown/podman-create.1.md.in

```diff
@@ -304,6 +304,8 @@ by having one container bind to localhost in the pod, and another connect to tha
 
 Suppress output information when pulling images
 
+@@option rdt-class
+
```

Comment:
are there man page references that we could add to this and the run man page?

Reply:
mh, nothing that I can think of at the moment.